bundles/org.openhab.binding.dmx/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -43,6 +43,10 @@
 		<label>Switch</label>
 		<description>Allows to switch a thing ON or OFF.</description>
 		<category>Light</category>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Light</tag>
```

Comment:
DMX usually controls lighting, it might also control smoke, fire or co2 effects when used by a stage setup. Maybe Power ?